src/hgvs/_data/hgvs.pymeta

```diff
@@ -122,7 +122,7 @@ pro_ident  = '='                             -> hgvs.edit.AARefAlt(ref='',alt=''
 
 # potentially indefinite/uncertain intervals
 c_interval = def_c_interval | '(' def_c_interval:iv ')' -> iv._set_uncertain()
-g_interval = def_g_interval | '(' def_g_interval:iv ')' -> iv._set_uncertain()
+g_interval = uncertain_g_interval:iv | ('(' def_g_interval:iv ')' -> iv._set_uncertain()) | def_g_interval
```

Comment:
I'm confused by this. The second alternative parse (with parentheses) is for an uncertain g_interval.